packages/server-wallet/src/models/__test__/channel.test.ts

```diff
@@ -100,3 +104,87 @@ describe('fundingStatus', () => {
     });
   });
 });
+
+const testChannelObj = TestChannel.create({aBal: 5, bBal: 3});
+let testChannel: Channel;
+let store: Store;
+
+function compareMilestones(milestoneStr: string[], milestoneNum: number[]) {
+  expect(milestoneStr.map(BN.from)).toEqual(milestoneNum.map(BN.from));
+}
+describe('Channel funding', () => {
+  beforeEach(async () => {
+    await DBAdmin.truncateDataBaseFromKnex(knex);
+
+    store = new Store(
+      knex,
+      defaultTestConfig().metricsConfiguration.timingMetrics,
+      defaultTestConfig().skipEvmValidation,
+      '0'
+    );
+  });
+
+  it('Funding milestones correct for A', async () => {
+    await testChannelObj.insertInto(store, {
+      participant: 0,
+      states: [0, 1],
+    });
+    testChannel = await Channel.forId(testChannelObj.channelId, knex);
+    compareMilestones(testChannel.fundingMilestones, [0, 5, 8]);
+  });
+
+  it('Funding milestones correct for B', async () => {
+    await testChannelObj.insertInto(store, {
+      participant: 1,
+      states: [0, 1],
+    });
+    testChannel = await Channel.forId(testChannelObj.channelId, knex);
+    compareMilestones(testChannel.fundingMilestones, [5, 8, 8]);
+  });
+
+  it('Check funding for A', async () => {
```

Comment:
nit but it might be nice to have slightly more descriptive test titles. Maybe something like?
```suggestion
  it('returns the correct funding status for the first participant', async () => {
```

Reply:
Thanks for pointing this out!